### DISCUSSION TITLE
RRC Release

### DISCUSSION THREAD

**Discussion by mohammadih** [srsran/srsRAN_Project#534] [General Help] [answered]
after building the docker and connecting gNB successfully, when I config the UE, both gnb and UE are USRP B210, after almost 5 seconds I got RRC release from UE side. If anyone knows how to solve this issue, I appreciate it sharing it with me
![Screenshot from 2024-03-28 09-53-52](https://github.com/srsran/srsRAN_Project/assets/60555990/35e042f2-6990-4539-b902-a969d5cd49fc)
![Screenshot from 2024-03-28 09-54-42](https://github.com/srsran/srsRAN_Project/assets/60555990/04af6be0-e574-4e04-8f56-1903a426b6c5)

**Comment by pgawlowicz** [comment, marked as answer]
@mohammadih please check here: https://github.com/srsran/srsRAN_Project/discussions/527#discussioncomment-8980792

**Reply by mohammadih** [reply]
Thanks, it seems this timeout is on 5 seconds by default. fixed
